sympy/integrals/manualintegrate.py

```diff
@@ -342,7 +342,10 @@ def _parts_rule(integrand, symbol):
     # log, inverse trig, algebraic (polynomial), trigonometric, exponential
     def pull_out_polys(integrand):
         integrand = integrand.together()
-        polys = [arg for arg in integrand.args if arg.is_polynomial(symbol)]
+        polys = [arg for arg in integrand.args if (arg.is_polynomial(symbol) or \
+                                                   (isinstance(arg, sympy.Pow) and \
```

Comment:
I feel like the noninteger and negative exponents should be lumped in with the polynomials. I interpret the 'A' in 'LIATE' as meaning algebraic expressions which is a larger set of expressions than polynomials. Having that check does not break any of the existing manualintegrate tests and it adds functionality. As opposed to breaking into multiple sections of code (which based on some experimentation could break some tests), I propose simply renaming pull_out_polynomials to pull_out_algebraic or something like that. In my opinion, that would be more consistent with the LIATE concept.